evaluator/src/main/resources/osadl/osadl.rules.kts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2022 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
```

Comment:
As we re-using work of OSADL I would like to give some credits e.g. link to their website where they make this dataset we use here available and add a note on the license of the JSON dataset.

Reply:
I believe such attribution should be better done as part of Markdown docs that explain how to use the evaluator and its built-in rules (which yet need to be created).

Reply:
IMO it should be in the source code as well OSADL is not known internationally that well and we as ORT maintainers should make clear on whose work we are building.

Reply:
I've added a code co0mment also to the rules, beyond what's already documented as part of the newly added `CompatibilityMatrix` class.